plugins/chat/assets/javascripts/discourse/components/chat-message-info.js

```diff
@@ -64,7 +34,36 @@ export default class ChatMessageInfo extends Component {
     if (user.groupModerator) {
       classes.push("is-category-moderator");
     }
-
+    if (user.new_user) {
+      classes.push("is-new-user");
+    }
+    if (user?.primary_group_name?.length) {
+      classes.push("group--" + user.primary_group_name);
+    }
     return classes.join(" ");
   }
+
+  get name() {
+    return this.prioritizeName
+      ? this.args.message?.get("user.name")
+      : this.args.message?.get("user.username");
+  }
+
+  get isFlagged() {
+    return (
+      this.args.message?.get("reviewable_id") ||
```

Comment:
We do `this.args.message?`, maybe it's better to just have:

```
get message() {
return this.args.message
}
```

and then it's shorter everywhere? There is some rule about these chains I forget what it's called tho.

Reply:
Same thing could be done with `this.args.user` IMO

Reply:
Yes not against this, can do :+1: